scripts/dev/run-test/fm-run-test

```diff
@@ -37,7 +37,7 @@ on_exit() {
 trap on_error ERR
 trap on_exit EXIT
 
-command time -q --format="$time_fmt" -o "$time_out_path" timeout -k 310 320 "$@" 2>&1 | ts -s > "$test_out_path"
+command time -q --format="$time_fmt" -o "$time_out_path" timeout -k 310 320 "$@" 2>&1 | ts -i "%.S" | ts -s "%M:%S" > "$test_out_path"
```

Comment:
Might be a bit noisy?

Reply:
We'll see.

We only print these when something fails and it's just 10 characters in the front.